source/_components/pi_hole.markdown

```diff
@@ -59,4 +64,24 @@ pi_hole:
   verify_ssl: false
 ```
 
+## Services
+
+The platform provides the following services to interact with your Pi-hole.
+
+### Service `pi_hole.disable`
+
+Disable your Pi-hole for the specified amount of time.
+
+| Service data attribute | Required | Type | Description |
+| ---------------------- | -------- | -------- | ----------- |
+| `duration` | `True` | timedelta | Time for which Pi-hole should be disabled | 
+
+_Note: This service requires `api_key` to be specified in the configuration._
+
+### Service `pi.hole_enable`
+
+Enable your Pi-hole.
+
+_Note: This service requires `api_key` to be specified in the configuration._
```

Comment:
@frenck ehm note? not in a div?

Reply:
Yes/no... if felt like adding big note boxes here would be overkill tbh

Reply:
I opted not to use a colored box as well because its an optional configuration setting.

Reply:
Alright no problem just a question 😉